develop/TOOLS/DIS/ToolWindows/DisValidatorToolWindow.md

```diff
@@ -6,14 +6,60 @@ uid: DisValidatorToolWindow
 
 If you click *Tool Windows > DIS Validator*, the *DIS Validator* window will appear.
 
-This window lists the results of the latest protocol validation in a tree structure, grouped by severity and category.
+This window lists the results of the latest protocol validation in a treegrid structure, grouped by severity and category.
 
-At the top of the tool window, you can find two buttons:
+At the top of the tool window, you can find the following buttons:
 
 - Click *Validate* to launch a new protocol validation.
-
 - Click *Export* to export the results of the latest validation to a CSV file.
+- Click the ![Show/hide suppressed results](~/develop/images/DisValidatorToolWindowToggleResultsIcon.png) icon to toggle visualization of the results to only show the Active items or all items (i.e., Active, Suppressed and Postponed items).
+
+> [!NOTE]
+> As of DIS v2.41, when you open a main DVE protocol and click *Validate*, DIS will not only validate the main protocol but also all its exported child DVE protocols.
+
+The validator results are listed in a treegrid which groups the validation results by their severity. The treegrid has the following columns:
 
-As of DIS v2.41, when you open a main DVE protocol and click *Validate*, DIS will not only validate the main protocol but also all its exported child DVE protocols.
+- Description: Description: Description of the issue
+- Information icon: This icon is shown in case more information is available about this validator check. Clicking the information icon will open a pop-up window which provided more details about the validator check.
+- Autofix icon: If this icon is displayed, an auto-fix is available. To execute the fix, from the context menu, select either *Fix* > *This error* or *Fix* > *All errors of this type*
+- State: Shows the state of this validation result (Active, Postponed or Suppressed). 
+- Certainty: Specifies whether the validator is certain or uncertain about the validity of the detected issue.
```

Comment:
Could be good here to refer to the fact that "Uncertain" results should be double checked by a developer and either fixed, either suppressed (with a link to the section about suppression lower on this page)